proto/frequenz/api/dispatch/fcr/prequalification.proto

```diff
@@ -27,4 +27,13 @@ message FcrPrequalificationSettings {
 
   // The SoC bound.
   optional SoCBound soc_bound = 3;
+
+  // Whether the initial SoC should be calculated or
+  // simply taken from the SoC bound.
+  //
+  // If true, the initial SoC for the operational phases is calculated
+  // based on the SoC bound and the power limit for the operational phase.
+  //
+  // If false, the initial SoC is set to the lower or upper SoC bound.
```

Comment:
lower or upper.. which one based on what?

Reply:
lower bound is used for charge test, upper bound is used for discharge test. otherwise it would not make sense unless you want to have your pq run fail instantly ;)

Reply:
It still would be good to clarify for the casual reader, not very familiar with how FCR prequalification works.

Also it would be nice to know in which circumstances you want to use one or the other? Maybe in the PR description is enough.

Reply:
Also, is there any relationship between `soc_bound` being optional and this `bool`? From this text it looks to me that the bounds are always used, so I wonder why are they optional.